src/components/molecules/Alert/Alert.tsx

```diff
@@ -56,31 +40,63 @@ export const Alert = ({
         }
     };
 
-    return isVisible ? (
+    const alertIcon = alertIconMapping[severity];
+
+    if (!isVisible) return null;
+
+    return (
         <section
             aria-label={severity}
             role='alert'
-            className={clsx('rounded-xl', {
-                'border-2 border-yellow bg-yellow/20': variant === 'outlined', // should be changed to handle different colors
-                'bg-[rgba(41, 45, 63, 0.60)] border border-white-10 shadow-tab':
-                    variant === 'solid',
-            })}
+            className={clsx(
+                'flex w-full flex-row items-start justify-between gap-1 rounded-md border px-2.5 text-2xs text-neutral-50 laptop:text-xs',
+                severityStyle[severity],
+                {
+                    'py-1.5': !subtitle,
+                    'py-2': subtitle,
+                }
+            )}
         >
-            <div
-                className={clsx(
-                    'flex w-full flex-row items-center justify-between gap-1 rounded-xl px-5 py-3',
-                    {
-                        'bg-gradient-to-b from-[rgba(111,116,176,0.35)] to-[rgba(57,77,174,0)]':
-                            variant === 'solid',
-                    }
+            <div className='flex items-start gap-1.5 pr-4 laptop:gap-2'>
+                {alertIcon && (
+                    <span>
+                        {cloneElement(alertIcon, {
+                            className: clsx(
+                                alertIcon.props.className,
+                                iconClass,
+                                'mt-[4px] laptop:mt-0.5'
```

Comment:
mt-1

Reply:
and it seems to me mt is same in both in mobile and desktop

Reply:
Yeah I thought it looked okay to reduce margin, but I tried again and it looks better with same top margin

Reply:
I used mt-[3px] instead. mt-1 (i.e. 4px) was simple too low compared to the title

Reply:
![image](https://github.com/Secured-Finance/secured-finance-app/assets/49697136/e78928b1-0525-4ed4-96d5-9692297e2c9a)
(20 -12 )/2 = 4px on top and bottom (without subt)
It means your icon is not of 12px 
with subt, it is even more clear that the margin top is 4px